### PR TITLE
add new parameter to use a new Localsearch algorithm to find an initial solution

### PR DESCRIPTION
**Pull request by Pierre-Graber**

### CODE REVIEW

File: api/v01/entities/vrp_input.rb [hunk 1]
```diff
@@ -152,6 +152,7 @@ module VrpConfiguration
   end
 
   params :vrp_request_preprocessing do
+    optional(:unconstrainted_initialization, type: Boolean, desc: 'Use experimental algorithm to find an initial solution')
```
Comment (alain-andre) [inline api/v01/entities/vrp_input.rb:155]: Why is it called `unconstrainted_initialization` when the PR calls about an initial solution to find ? 
Shouldn't it be called `initialise_solution` ?

File: config/environments/development.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@
 require './wrappers/demo'
 require './wrappers/vroom'
 require './wrappers/ortools'
+require './wrappers/localsearch'
```
Comment (alain-andre) [inline config/environments/development.rb]: Shouldn't all modifications in `development.rb` be done in `production.rb` too?

File: wrappers/localsearch.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+require './wrappers/wrapper'
+require './wrappers/ortools_vrp_pb'
+require './wrappers/ortools_result_pb'
+module Wrappers
+  class Localsearch < Ortools
```
Comment (alain-andre) [inline wrappers/localsearch.rb]: I'm wondering about the name `Localsearch`. 

This class is called `Localsearch`, is used if `unconstrainted_initialization` is true and uses a `initial_solution`. 

Is there not a misunderstanding in the names?

File: api/v01/entities/vrp_input.rb (hunk 1 above)
Comment (guillaumemaran) [reply api/v01/entities/vrp_input.rb:155]: There are many initial solution calculation methods. This one is just a different one, which is unconstrainted... But you right, we should call it this way everywhere.

File: config/environments/production.rb [hunk 4]
```diff
@@ -64,6 +64,7 @@ module OptimizerWrapper
       demo: DEMO,
       vroom: VROOM,
       ortools: ORTOOLS,
+      unconstrainted_initialization: UNCONSTRAINTED_INITIALIZATION,
```
Comment (alain-andre) [inline config/environments/production.rb:68]: The initialization of the variable is missing I think

`UNCONSTRAINTED_INITIALIZATION = Wrappers::UnconstraintedInitialization.new(tmp: TMP_DIR)`

**Review by alain-andre — APPROVED** [review]

**Pull request merged**